src/SWP/Bundle/CoreBundle/Resources/config/services.yml

```diff
@@ -764,7 +766,7 @@ services:
             - '@swp.factory.article_statistics'
             - '@swp.object_manager.article'
 
-    Sylius\Bundle\ThemeBundle\HierarchyProvider\ThemeHierarchyProviderInterface: '@sylius.theme.hierarchy_provider'
+#    //Sylius\Bundle\ThemeBundle\HierarchyProvider\ThemeHierarchyProviderInterface: '@sylius.theme.hierarchy_provider'
```

Comment:
why is this commented out? Since the class is autowired this alias can be removed